docs/setup/settings.asciidoc

```diff
@@ -29,6 +29,27 @@ service that Kibana uses to display map tiles in tilemap visualizations. By defa
 `tilemap.options.attribution:`:: *Default: `"© [Elastic Tile Service](https://www.elastic.co/elastic-tile-service)"`* The map attribution string.
 `tilemap.options.subdomains:`:: An array of subdomains used by the tile service.
 Specify the position of the subdomain the URL with the token `{s}`.
+
```

Comment:
Should "WGS84 coordinate reference" be "WGS84 coordinate reference system"? 

The other descriptions are init capped. I'd also make some other minor edits:

Specifies additional vector layers for use in <<regionmap, Region Map>> visualizations.  Each layer object points to an external vector file that contains a `geojson FeatureCollection`. The file must use the WGS84 coordinate reference system and only include polygons. If the file is hosted on a separate domain from Kibana, it needs to be CORS-enabled so Kibana can download the file. The `url` field also serves as a unique identifier for the file. Each layer can contain multiple fields to indicate what properties from the geojson features you want to expose. The `field.description` is the human readable text that is shown in the Region Map visualization's field menu. The following example shows a valid `regionmap` configuration.